docs/install_apt_debian.md

```diff
@@ -4,10 +4,11 @@ title: "Installation: Debian"
 
 # Debian Installation
 
-Caffe packages are available for `Debian/unstable`. Debian/stable users
-should take a look at Ubuntu installation instruction.  
+Caffe packages are available for `Debian/unstable` (Debian 10). Debian/stable
```

Comment:
I think there is no need to refer `Debian/unstable` such a rolling release as `Debian 10`. And actually Debian/unstable is currently Stretch (Debian 9).